test/functional/p2p_orphan_handling.py

```diff
@@ -41,6 +42,8 @@
 # for one peer and y seconds for another, use specific values instead.
 TXREQUEST_TIME_SKIP = NONPREF_PEER_TX_DELAY + TXID_RELAY_DELAY + OVERLOADED_PEER_TX_DELAY + 1
 
+MAX_ORPHANS = 100
```

Comment:
nit: Could put it in `test/functional/test_framework/mempool_util.py` as `DEFAULT_MAX_ORPHAN_TRANSACTIONS` to match https://github.com/bitcoin/bitcoin/blob/947f2925d55f363cf1880315d467fc2edac04545/src/net_processing.h#L27

Reply:
Thanks! Updated in [5c299ec](https://github.com/bitcoin/bitcoin/pull/31040/commits/5c299ecafe6f336cffa145d28036b04b87e26712)